lib/elixir/pages/cheatsheets/enum-cheat.cheatmd

```diff
@@ -1006,9 +1006,10 @@ See `Enum.zip_with/2` for zipping many collections at once.
 iex> fruits = ["apple", "banana", "orange"]
 iex> counts = [3, 1, 6]
 iex> Enum.zip_reduce(fruits, counts, 0, fn fruit, count, acc ->
-...>   {{fruit, count + acc}, acc + count}
+...>   price = if fruit == "banana", do: 1, else: 2
+...>   acc + price * count
 ...> end)
-[{"apple", 3}, {"banana", 4}, {"orange", 10}]
+19
```

Comment:
This example was broken, failing with
```elixir
** (ArithmeticError) bad argument in arithmetic expression: 1 + {{"apple", 3}, 3}
```

I've been trying to figure out an alternative, but feel free to propose a better one.

Reply:
Ah, thanks! And here I was thinking it was inconsistent... :) This makes me happy!